### DISCUSSION TITLE
StackOverflow tags: avalonia vs avaloniaui ?

### DISCUSSION THREAD

**Discussion by moon6969** [AvaloniaUI/Avalonia#7843] [General]
Stack Overflow has both [avalonia](https://stackoverflow.com/tags/avalonia) and [avaloniaui](https://stackoverflow.com/tags/avaloniaui) tags...

Both have the same description:
> AvaloniaUI (aka Avalonia) is a multiplatform GUI-Framework using XAML for the creation of GUIs. It has many similarities to WPF and can run on Windows, Linux, macOS, Android and iOS. 

Is there any difference? If none, should they be [merged or synonymised](https://meta.stackexchange.com/questions/70710/what-are-tag-synonyms-and-merged-tags-how-do-they-work)?

**Comment by jmacato** [comment]
they should be synonymised yes

**Comment by robloo** [comment]
What is the history behind these terms? Which is more current and correct?

My assumption is AvaloniaUI is older than just Avalonia. I noticed 'UI' was removed from the website branding as well.

**Reply by jmacato** [reply]
yeah just Avalonia from now on iirc, we havent had any too much discussions about branding lately though

**Comment by moon6969** [comment]
I don't have reputation (or moral authority) to suggest synonyms on Stack Overflow.
I believe the suggestion can be made at https://stackoverflow.com/tags/avalonia/synonyms by...
> Users with more than 2500 reputation and a total answer score of 5 or more on the tag

Could one of you do this? Or should I raise an Avalonia issue here on github?
